rq_scheduler/scheduler.py

```diff
@@ -151,9 +161,11 @@ def enqueue_in(self, time_delta, func, *args, **kwargs):
         job_id = kwargs.pop('job_id', None)
         job_ttl = kwargs.pop('job_ttl', None)
         job_result_ttl = kwargs.pop('job_result_ttl', None)
+        job_description = kwargs.pop('job_description', None)
 
         job = self._create_job(func, args=args, kwargs=kwargs, timeout=timeout,
-                               id=job_id, result_ttl=job_result_ttl, ttl=job_ttl)
+                               id=job_id, result_ttl=job_result_ttl, ttl=job_ttl,
+                               description=job_description)
```

Comment:
Can we please rename this to `description`?

Reply:
Every single other job-related kwarg is prefixed with `job_` -- are you sure? I worry that having `description` be different may be confusing.

Reply:
I was thinking that we should bring the argument names closer to RQ's over time. But you're right, this will look inconsistent with other argument's naming scheme. We'll keep this for now.